src/bundle/Resources/public/js/CKEditor/embed/image/embed-image-toolbar.js

```diff
@@ -8,7 +8,7 @@ class IbexaEmbedImageToolbar extends Plugin {
 
     getSelectedEmbedImageWidget(selection) {
         const viewElement = selection.getSelectedElement();
-        const isEmbedImage = viewElement && viewElement.hasClass('ez-embed-type-image');
+        const isEmbedImage = viewElement && viewElement.hasClass('ibexa-embed-type-image');
```

Comment:
```suggestion
        const isEmbedImage = viewElement?.hasClass('ibexa-embed-type-image');
```
prophet of optional chaining has come...